readme.md

```diff
@@ -81,8 +85,8 @@ var Panda = function(name, age) {
 ```
 
 Your Answer:
-```text
-
+```
+	N/A
 ```
```

Comment:
Would love to see you make an attempt at this, even you don't know the syntax, psuedo code the steps. Would recommend you review the lesson plan, and then come back to this.

Anytime I hear "add an instance method for all pandas called eat_bamboo", that would signify I want to add a new method to the prototype of a panda.